coderedcms/views.py

```diff
@@ -39,49 +36,35 @@ def search(request):
         search_query = search_form.cleaned_data['s']
         search_model = search_form.cleaned_data['t']
 
-        # get all codered models
-        pagemodels = sorted(get_page_models(), key=lambda k: k.search_name)
-        # get filterable models
+        # get all page models
+        pagemodels = sorted(get_page_models(), key=lambda k: k.__name__)
+        # If any types have the search_filterable flag set, only show those
```

Comment:
The filtering can continue to work as it used to. This is only used to show additional tabs in the UI. If it is empty, that's fine.

Reply:
The purpose of this change was to allow for sorting based on Wagtail models as well, in the case that CRX is added after initial work is done, while still allowing users to hide undesired pages if necessary.

Reply:
So search_filterable is not supposed to behave how your comment indicates (only show those). Instead it supposed to only present the option of filtering by those models in the template. It does not limit the main search to those models. Just presents the option to filter by them.

The variable name should probably be renamed "filterable_pagetypes" or something more descriptive rather than just "pagetypes"